From 8f7501b06c1e0768131ae939b7b5e178074e02c7 Mon Sep 17 00:00:00 2001
From: Guillaume Papin <guillaume.papin@epitech.eu>
Date: Thu, 8 Mar 2018 01:06:38 +0100
Subject: [PATCH 2/2] CMake: remove extra dependencies curses, backatrace, z,
 pthread

None of these features seems to be used by IWYU,
this breaks the conda build because the dependencies aren't specified.
I believe, we should "link what we use", so if it's not used, remove it.
---
 CMakeLists.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7aad3bd..4724eae 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -173,14 +173,7 @@ if( WIN32 )
     version # For clangDriver's MSVCToolchain
   )
 elseif( UNIX )
-  include(FindCurses)
-  include(FindBacktrace)
-
   target_link_libraries(include-what-you-use
-    pthread
-    z
-    ${Backtrace_LIBRARIES}
-    ${CURSES_LIBRARIES}
     ${CMAKE_DL_LIBS}
   )
 else()
-- 
2.10.0

